lesson14.html

```diff
@@ -0,0 +1,161 @@
+        }
+      }
+      .tooltiptext {
+        visibility: hidden;
+        opacity: 0;
+        transition: opacity 1s ease, visibility 1s ease;
+        background-color: gray;
+        color: black;
+        text-align: center;
+        border-radius: 9px;
+        position: absolute;
+        z-index: 1;
+      }
+      .tooltip1 .tooltiptext {
+        top: 50%;
+        width: 120px;
+        padding: 5px 10px;
+        left: 0;
+        transform: translate(-15%, -140%);
+      }
+      .tooltip1:hover .tooltiptext,
+      .tooltip2:hover .tooltiptext,
+      .tooltip3:hover .tooltiptext,
+      .tooltip4:hover .tooltiptext {
+        visibility: visible;
+        opacity: 1;
+      }
+      .tooltip1 .tooltiptext::after,
+      .tooltip2 .tooltiptext::after,
+      .tooltip3 .tooltiptext::after,
+      .tooltip4 .tooltiptext::after {
+        content: " ";
+        position: absolute;
+      }
+      .tooltip1 .tooltiptext::after,
+      .tooltip3 .tooltiptext::after {
+        border-left: 10px solid transparent;
+        border-right: 10px solid transparent;
+        transform: translateX(-50%);
+      }
+      .tooltip1 .tooltiptext::after {
+        top: 100%;
+        left: 50%;
+        border-top: 10px solid gray;
+      }
+      .tooltip2 .tooltiptext {
+        top: -50%;
+        width: 120px;
+        padding: 9px;
+        left: 0;
+        transform: translateX(77%);
+      }
+      .tooltip2 .tooltiptext::after,
+      .tooltip4 .tooltiptext::after {
+        top: 50%;
+        border-bottom: 10px solid transparent;
+        border-top: 10px solid transparent;
+        transform: translateY(-50%);
+      }
+      .tooltip2 .tooltiptext::after {
+        left: -10px;
+        border-right: 10px solid gray;
+      }
+      .tooltip3 .tooltiptext {
+        top: 110%;
+        width: 110px;
+        padding: 10px 20px;
+        left: 50%;
+        transform: translateX(-50%);
+      }
+      .tooltip3 .tooltiptext::after {
+        top: -10%;
+        left: 80px;
+        border-bottom: 10px solid gray;
+      }
+      .tooltip4 .tooltiptext {
+        transform: translateY(-50%);
+        top: 50%;
+        right: 109%;
+        width: 120px;
+        padding: 9px;
+      }
+      .tooltip4 .tooltiptext::after {
+        left: 100%;
+        border-left: 10px solid gray;
+      }
+    </style>
+  </head>
+  <body>
+    <p>
+      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Et, perferendis!
+      Repudiandae deleniti, nostrum blanditiis impedit laudantium, numquam atque
+      itaque illo facere odio ut tempore quibusdam obcaecati, recusandae minus
+      corporis rem!
+    </p>
+    <p>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam voluptatum
+      nemo quo ullam deleniti rem maiores a dolor maxime praesentium minus earum
+      optio sapiente voluptatibus facere, blanditiis ea tempore facilis!
+    </p>
+    <div class="container_blue">
+      <div class="trigger tooltip1">
```

Comment:
would be more logic to name the common class - "tooltip" instead of "trigger"
So you will have 4 tooltips:
```
     .tooltip.tooltip1
     .tooltip.tooltip2
     .tooltip.tooltip3
     .tooltip.tooltip4